### DISCUSSION TITLE
Is LibreLingo still active?

### DISCUSSION THREAD

**Discussion by bsciretti** [kantord/LibreLingo#2661] [Q&A]
Hello, I'm a member of the Academia Bonvesin de la Riva, which is an entity devoted to promoting and studying the Lombard language. I'm interested in creating a course with LibreLingo, but I found some problem in the documentation. Is the project still active and, if so, can I start working on the course? Thanks!

**Comment by kantord** [comment]
yes, LibreLingo is still active. At this stage of the project there are still some limitations, but you can definitely create your course.

Here's an example of what you might be able to achieve in the current state: https://librelingo.app/course/basque-from-english/

I am mainly working on development and coordinating the project, so I'm not even the most experience person about creating courses at this point. You can join our chat in order to talk to people who are involved in creating courses and have experience with it: https://app.element.io/#/room/#space-librelingo:matrix.org
